assembl/tests/views_tests/test_api2.py

```diff
@@ -899,3 +900,54 @@ def test_add_timeline_event(test_app, discussion):
     # check that the link was made in both directions
     assert phase1_data['next_event'] == phase2_data['@id']
     assert phase1_data['@type'] == 'DiscussionPhase'
+
+
+def test_phase1_export(proposals_with_sentiments, discussion, test_app):
+
+    THEMATIC_NAME = 0
+    QUESTION_ID = 1
+    QUESTION_TITLE = 2
+    POST_BODY = 3
+    POST_LIKE_COUNT = 4
+    POST_DISAGREE_COUNT = 5
+    POST_CREATOR_NAME = 6
+    POST_CREATOR_EMAIL = 7
+    POST_CREATION_DATE = 8
+    SENTIMENT_ACTOR_NAME = 9
+    SENTIMENT_ACTOR_EMAIL = 10
+    SENTIMENT_CREATION_DATE = 11
+
+    response = test_app.get(
+        '/data/Discussion/{}/phase1_csv_export'.format(discussion.id))
+    csv_file = BytesIO()
+    csv_file.write(response.app_iter[0])
+    csv_file.seek(0)
+    assert response.status_code == 200
+    result = csv.reader(csv_file, dialect='excel')
```

Comment:
Please put a comment on this test that the content of this csv is coming from the fixtures of thematics, sentiments, posts, etc. And that if any changes are made to those fixtures, this test must also be changed.